src/Collection/Collection.router.spec.ts

```diff
@@ -0,0 +1,62 @@
+import supertest from 'supertest'
+import { v4 as uuidv4 } from 'uuid'
+import { Collection } from './Collection.model'
+import { app } from '../server'
+import {
+  wallet,
+  createAuthHeaders,
+  buildURL,
+  mockExistsMiddleware,
+  mockAuthorizationMiddleware,
+} from '../../spec/utils'
+
+const server = supertest(app.getApp())
+jest.mock('./Collection.model')
+
+describe('Collection router', () => {
+  const collectionAttributes = {
+    id: uuidv4(),
+    name: 'Test',
+    eth_address: wallet.address,
+    salt: '',
+    contract_address: '',
+    is_published: false,
+    is_approved: false,
+    minters: [],
+    managers: [],
+    forum_link: null,
+    lock: null,
+    reviewed_at: new Date(),
+    created_at: new Date(),
+    updated_at: new Date(),
+  }
+
+  describe('when locking a Collection', () => {
+    it('should update the lock with .now() on the supplied collection id for the owner', async () => {
+      jest.useFakeTimers()
+
+      mockExistsMiddleware(Collection, collectionAttributes.id)
+      mockAuthorizationMiddleware(
+        Collection,
+        collectionAttributes.id,
+        wallet.address
+      )
```

Comment:
I would move everything related to the setup of the context to a `beforeEach` in order to leave just the execution and the assertions inside the `it`.